src/schema.rs

```diff
@@ -560,17 +560,17 @@ pub struct FlowTable {
     /// The *devices* are specified as iifname(s) of the input interface(s) of the traffic that should be offloaded.
     ///
     /// Devices are required for both traffic directions.
-    /// Vec of device names, e.g. `vec!["wg0".to_string(), "wg0".to_string()]`.
-    pub dev: Option<Vec<String>>,
+    /// Cow slice of device names, e.g. `vec!["wg0".into(), "wg1".into()].into()`.
```

Comment:
nice!

Reply:
I am afraid of the impact of the breaking changes for users of this crate, but this is obviously the way forward.